andersen-homework/src/main/java/org/util/BusTicketValidator.java

```diff
@@ -0,0 +1,93 @@
+package org.util;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.model.Ticket;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BusTicketValidator {
+    private int totalTickets = 0;
+    private int validTickets = 0;
+    private Map<String, Integer> violationCounts = new HashMap<>();
+
+    public BusTicketValidator() {
+        violationCounts.put("start date", 0);
```

Comment:
Also I see that "start date" duplicated many times here, let's create constant for such values

Reply:
Add please constants for duplicated values such as "start date", "ticket type", "price"

Reply:
have created constant